src/components/modals/AddressModal.js

```diff
@@ -234,7 +234,7 @@ class AddressModalConnected extends React.Component<Props, State> {
         const publicAddress = await checkPubAddress(fioPlugin, uri.toLowerCase(), coreWallet.currencyInfo.currencyCode, currencyCode)
```

Comment:
```
const { public_address: publicAddress } = await fioPlugin.otherMethods.getConnectedPublicAddress(fioAddress.toLowerCase(), chainCode, tokenCode)
      if (publicAddress && publicAddress.length > 1) {
        return publicAddress
      }
```
make check for fio address in `getConnectedPublicAddress`, and add check for empty public address, throw different errors with different codes